config/dev-ssriche.env

```diff
@@ -0,0 +1,2 @@
+export GLOBAL_CUSTOM_DEPLOYMENT_NAME_IDENTIFIER="ssriche"
```

Comment:
I am thinking that these would be good in the examples dir....rather than local development